pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -59,6 +60,11 @@ const (
 	maxDNSSearchListChars = 256
 )
 
+const (
+	multiQueueMinQueues = 2
+	multiQueueMaxQueues = 256
```

Comment:
The upper limit sounds great.

Reply:
Discussed this with @vladikr. Nowadays VMs can have more cores than 256, but only 256 queues. It is still expected that a VM with 512 cores and 256 queues has more benefits than a VM with 1 queue and 512 cores, so I don't think that we need a limit. We should just limit ourselves to 256 queues if more than 256 vCPUs are given.

Reply:
Will update the PR to simply cap the number of queues to the max number of tap device queues then.

Thanks for clearly defining the proper behavior.